ocpp_codec/validators.py

```diff
@@ -63,6 +63,14 @@ def noop(value: typing.Any) -> typing.Any:
     return value
 
 
+def min_length(length: int, value: typing.Any) -> typing.Any:
+    """Validates that an input doesn't go below a given length."""
+    actual_length = len(value)
+    if actual_length < length:
+        raise errors.PropertyConstraintViolationError("Input is too short", min_length=length)
```

Comment:
Maybe add the input and/or `actual_length` to the log so we know exactly why that went wrong?

Reply:
It seems required list fields must have at least one element to be considered defined.
As such, we do not need this validator anymore so I just removed it.
I took into account your comment and applied it to `max_length` validator which has the same issue.